src/Uno.Templates/content/unoapp/MyExtensionsApp._1/MainPage.xaml.cs

```diff
@@ -7,7 +7,8 @@ public MainPage()
 	{
 //+:cnd:noEmit
 #if useCsharpMarkup
-		this.Content(new StackPanel()
+		this.Background(Theme.Brushes.Background.Default)
```

Comment:
We need to update where we have `Theme.Brushes`... as it only works for Material

Reply:
Updated